### PR TITLE
Updated README to include new TSD plugin Interface

### PR DESCRIPTION
**Pull request by ryryog25**
Updated to include new description of the plugin and setup process.

### CODE REVIEW

**Comment by ryryog25** [comment]
The only thing left is that the image below the instructions should be moved and should also be updated, I can do that now...

**Comment by ryryog25** [comment]
Ok, I got the images formatted correctly, had to do some trial and error (I probably did it wrong, don't really know how to use GitHub.)

**Comment by ryryog25** [comment]
I think that I have made all the changes that there should be, if there is anything else tell me!

File: README.md [hunk 1]
```diff
@@ -109,25 +109,50 @@ That's it! You now have a fully-functional The Spaghetti Detective server that y
 
 # Configure The Spaghetti Detective OctoPrint Plugin to use your own server
 
-Before you can configure The Spaghetti Detective OctoPrint Plugin to use your own server, you need add a printer to The Spaghetti Detective server you just built and obtain the secret token for that. To do so:
+Before you can configure The Spaghetti Detective OctoPrint Plugin to use your own server, you need add a printer to The Spaghetti Detective server you just built and obtain the 6-digit Verification Code for that. To do so:
 
 1. Pointing your browser to `http://your_server_ip:3334`.
 
-1. Log in as a user (you can just login with `root@example.com` but it's more secure to use a non-admin user).
+2. Log in as a user (you can just login with `root@example.com` but it's more secure to use a non-admin user).
```
Comment (kennethjiang) [inline README.md:116]: Using `1.` is markdown's way to say these are numbered bullets. They will be displayed as the correct sequence numbers. The advantage of not hard-coding the sequence number is we don't have to change all the bullets if we decide to insert a line.

File: README.md [hunk 2]
```diff
@@ -109,25 +109,50 @@ That's it! You now have a fully-functional The Spaghetti Detective server that y
...
+1. Open the OctoPrint Settings and navigate to "Access Anywhere - The Spaghetti Detective"
+
+2. Click "Settings"
+
+3. Find and change the Server Address to `http://your_server_ip:3334/` (use https:// if you have it, if you aren't sure, just use http://) you MUST include the "http://" and trailing "/"
 
 4. Click "Save". OctoPrint isn't necessarily smarter than Django after all.
 
-![Site configuration](https://raw.githubusercontent.com/TheSpaghettiDetective/TheSpaghettiDetective/master/docs/plugin_config.png)
+<img src="https://github.com/ryryog25/TSD-fork-resources/blob/main/Screen%20Shot%202021-03-02%20at%209.20.58%20AM.png?raw=true" width="500">
```
Comment (kennethjiang) [inline README.md:136]: Can you add the images as part of your PR? It's usually a good idea to include everything needed to be in the same repo to make it self-contained.

Comment (ryryog25) [reply README.md:136]: Sure!

File: README.md (hunk 1 above)
Comment (ryryog25) [reply README.md:116]: Ok, so I should make all of them one?

**Comment by ryryog25** [comment]
I can no longer edit the file, going to redo with just one pull request for everything.

**Pull request closed without merging**